extensions/sftp/tests/FetchSFTPTests.cpp

```diff
@@ -142,7 +142,7 @@ class FetchSFTPTestsFixture {
     std::fstream file;
     std::stringstream ss;
     ss << src_dir << "/vfs/" << relative_path;
-    utils::file::FileUtils::create_dir(utils::file::FileUtils::get_parent_path(ss.str()));  // TODO
+    utils::file::FileUtils::create_dir(utils::file::FileUtils::get_parent_path(ss.str()));  // TODO(_)
```

Comment:
I doubt anyone knows what this TODO refers to, so I think we should remove it rather than make it cpplint-compliant

also, `utils::file::FileUtils::` is the same as `utils::file::`, so we can remove `FileUtils`; maybe that is what the TODO was about?

```suggestion
    utils::file::create_dir(utils::file::get_parent_path(ss.str()));
```